common/lib/capa/capa/templates/clarification.html

```diff
@@ -0,0 +1,4 @@
+<span class="clarification">
```

Comment:
@bradenmacdonald, The code above is exactly right. The aria-role [adds no behavior](http://www.w3.org/TR/aria-in-html/#role-not) to the span though.  It is there to communicate to the screen reader user that this block of content has a specific role ([roles](http://www.w3.org/TR/wai-aria/roles#role_definitions) are enumerated, so we can't just make them up unfortunately :) .   You would need to add the show/hide behavior on focus with JS.